student.rb

```diff
@@ -1,9 +1,11 @@
 require_relative 'person'
 
 class Student < Person
-  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+  attr_accessor :classroom
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
     super(age, parent_permission: parent_permission, name: name)
-    @classroom = classroom
+    @classroom = nil
   end
 
```

Comment:
- [ ] Lastly, In the `student.rb` file, you need to ensure that there is a `belongs-to relationship` between `Classroom` and `Student` :+1:  You did an incredible job creating the `'has-many' side (a classroom has many students)` so kindly create a method that serves as the `'belongs-to' side (a student belongs to a classroom)` :wink: 
- [ ] That way, when setting the classroom for a student it also adds it to the classrooms' students :+1: 

- Example implementation 
```ruby
class Student < Person
  attr_reader :classroom
  
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```